spec/classes/init_spec.rb

```diff
@@ -243,7 +240,16 @@
         it { is_expected.to contain_file(global_conf_file).with_content(%r{input\(type="imtcp" port="514"\)}) }
         it { is_expected.to_not contain_file(global_conf_file).with_content(/^\$DefaultNetStreamDriver/) }
       end
+
+      context "including the rsyslog.d directory" do
+        let(:hieradata) { 'include_rsyslog_d' }
+	it {
+          is_expected.to contain_rsyslog__rule('15_include_default_rsyslog/include_default_rsyslog.conf')
+#            .with_content(/\$IncludeConfig \/etc\/rsyslog.d\/\*\.conf\\n$/)
```

Comment:
It doesn't seem like this should be commented out.

Reply:
also fixed